required/amsmath/changes.txt

```diff
@@ -2,6 +2,12 @@
 All changes above are only part of the development branch for the next release.
 ================================================================================
 
+2024-08-02  Frank Mittelbach  <Frank.Mittelbach@latex-project.org>
+
+	* amsmath.dtx (section{Ellipsis dots}):
+	Support special \dots magic in case of robust characters via \protected
```

Comment:
Extend the . . . to cover the case of ...

Reply:
don't think so: it  supports 2 cases: the one using \protected\def and the one with the symbol being defined with \DeclareRobustcommand. Tried a different rewrite.